integration-tests/requirements.txt

```diff
@@ -4,14 +4,19 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
+# To install pip-compile, run:
+#    pip install pip-tools
+#
```

Comment:
- since this requirements.txt is generated (by the pip-compile) your note will most likely be lost with some future update
- we use python3 and pip3, so the correct command is `pip3 install pip-tools`